### PR TITLE
Run2-alca90 Add a missing collection for Muons in HB/HE

### PR DESCRIPTION
**Pull request by bsunanda**

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
The change in event content increases data volume by 6% - the new size requirement is roughlt 0.85 MB per event stored.

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@arunhep @lpernie Can you review and approve this PR?

**Comment by lpernie** [comment]
+1

**Comment by lpernie** [comment]
Hi @bsunanda, could you summarize why the need of these muon collections? thanks.

**Comment by davidlange6** [comment]
hi @lpernie did you really mean to +1 this PR? Perhaps your questions should be answered first?

**Comment by bsunanda** [comment]
We have tested earlier that we need all these muon collections to make a selection of good muon candidates downstream for the final calibration. This was answered earlier in AlCaDb meetings. The only change from earlier time is to include pre-rechits. This collection comes from new strategy of HCAL reconstruction and one needs to work with uncollapsed as well as collapsed RecHits. This addition has increased the data volume requirement by 6%. The muon collection requirement has been there from beginning of 2016.

**Comment by lpernie** [comment]
Hi @davidlange6 , I'm sorry for inverting the order, I just wanted a confirmation (next time I will wait).

**Comment by davidlange6** [comment]
merge
